config/overlays/openshift/controller_manager_config.yaml

```diff
@@ -30,6 +30,7 @@ featureGates:
     openshiftRoute: true
     servingCertsService: true
   prometheusOperator: true
+  grafanaOperator: true
```

Comment:
```suggestion
  grafanaOperator: false
```

The operator is not installed by default on OpenShift.

Reply:
Will change it.